front/src/app/pages/home-page/home-page.component.css

```diff
@@ -26,23 +28,29 @@ body {
   justify-content: center;
   width: 30%;
   top: 40vh;
-  position: relative;
 }
 
 h4 {
   font-size: 2em;
-  position: sticky;
   top: 25vh;
   color: white;
+  padding-top: 15%;
 }
 
 app-leaderboard {
   display: flex;
   align-items: center;
-  position: relative;
-  top: 100vh;
   left: 0;
   right: 0;
   bottom: 0;
   padding-bottom: 5vh;
 }
+
+app-game-summary-list {
+  display: flex;
```

Comment:
Make it so the history is hidden on page's first load like the leaderboard was using `top: 100vh` ?

Reply:
I can't find a way to do this :/